### ISSUE TITLE
Module parse failed: Unexpected token /dist/index.js (create-react-app V4)

### ISSUE THREAD

**Issue by ivankoop** [FlatFilers/sdk#83]
Hi guys,

Im currently trying to build my create-react-app application using `@flatfile/sdk v2.0.4`, but im facing the following issue:

```
./node_modules/@flatfile/sdk/dist/index.js 401:359
Module parse failed: Unexpected token (401:359)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|         class v extends i {
|           constructor(e, t) {
>             super(e), r(this, "code", "FF-UA-00"), r(this, "name", "UnauthorizedError"), r(this, "debug", "The JWT was not signed with a recognized private key or you did not provide the necessary information to identify the end user"), r(this, "userMessage", "There was an issue establishing a secure import session."), e && (this.debug = e), this.code = t ?? "FF-UA-00";
|           }
|
```

The issue might be on `?? "FF-UA-00";`
The nullish coalescing ( ?? ) operator is an ES2020 feature

Looks like ES2020  was emitted to the `/dist` folder so my cra babel loader is not able to parse it,
in order to fix it I need to include the path on my webpack config.

**Comment by emersonthis** [comment]
We're seeing the same issue. Just tried updating to v2.1.5 and it's still happening.

**Comment by emersonthis** [comment]
@ivankoop Can you elaborate a bit on your workaround?

**Comment by ivankoop** [comment]
Hi @emersonthis , sure
using `customize-cra` on `config-overrides.js` you can include `node_modules/@flatfile`

```
const { override, babelInclude } = require('customize-cra');
const fs = require("fs");

module.exports = (config, env) => { 
  return Object.assign(
    config,
    override(
      babelInclude([fs.realpathSync("node_modules/@flatfile")])
    )(config, env)
  ); 
}

```

**Comment by dboskovic** [comment]
We have a fix up that will be merged later today as version `2.2.0` of the SDK. You can test with `2.2.0-rc0` immediately
